### ISSUE TITLE
just wondering

### ISSUE THREAD

**Issue by MilkSugarCoffee** [vdev99/Basic-Seal-.NET-Licensing-System#2]
yeah so i got this working and everything but is this made for localhost only? like pcs in same area

cause obv it doesnt work when someone else uses the program it wont connect.

anyway to make it work outside my network? tried using mysql online etc but didnt really work, mainly cuz database was made in mysql 8.

**Comment by MilkSugarCoffee** [comment]
What do i need to do to make this working online and be able to connect to the website online and program licensing works etc.

**Comment by vdev99** [comment]
Yes, the project is configured to run locally, and yes it could work outside your home network with correct setup.
But as of now, the project is not really production ready at this state as it is stated in the readme, and is more like a project to take inspiration from, or a project to build on.

![image](https://user-images.githubusercontent.com/86075693/123514537-ca72f480-d693-11eb-86ea-37bf72d45bad.png)

I plan to do updates on the following weeks/months to make it more usable.

In this version, the website and the backend would both need a lot of configuration to make it work in production, also, the blazor website and the backend are separated and both has its own process.

You would need a mysql remote database, or a local database on the computer where the backend will be installed, and you also have to configure the backend projects appsettings.json according to it.
I would recommend a VPS with linux with minimum 1GB of ram with installed .NET Core 3.1.
You also need a SSL certificate with which you need to configure the frontend and backend projects for secure communication.
I cant really guide you on this, as this version was not intended for production.
